djangoproject/tests.py

```diff
@@ -164,3 +166,40 @@ def test_single_h1_per_page(self):
                     response = self.client.get(url)
                     self.assertEqual(response.status_code, 200)
                     self.assertContains(response, "<h1", count=1)
+
+
+class SecurityTxtFileTests(TestCase):
+    """
+    Tests for the security.txt file.
+    """
+
+    def test_security_txt_not_expired(self):
+        """
+        The security.txt file should not be expired.
+        """
+        FILE_PATH = settings.BASE_DIR / ".well-known" / "security.txt"
+        with open(FILE_PATH) as f:
+            content = f.read()
+            # Read the line that starts with "Expires:", and parse the date.
+            for line in content.splitlines():
+                if line.startswith("Expires:"):
+                    expires = line.strip("Expires: ")
+                    break
+            else:
+                self.fail("No Expires line found in security.txt")
```

Comment:
Perfect use of a `for ... else` 👏🏻  However I think a regex perfect for this kind of situation:
```suggestion
match = re.search("^Expires: (.*)$", content, flags=re.MULTILINE)
            if match is None:
                self.fail("No Expires line found in security.txt")
            else:
                expires = match[1]
```

Reply:
TIL `for ... else`.  🤯 🐍 ❤️ 
Thank you. 🫶